adapters/repos/db/shard_shutdown.go

```diff
@@ -44,6 +45,17 @@ import (
 // component was initialized. If not, it turns it into a noop to prevent
 // blocking.
 func (s *Shard) Shutdown(ctx context.Context) (err error) {
+	start := time.Now()
+	defer func() {
+		s.index.metrics.ObserveUpdateShardStatus(storagestate.StatusShutdown.String(), time.Since(start))
+
+		if err != nil {
+			return
+		}
+
+		s.UpdateStatus(storagestate.StatusShutdown.String())
```

Comment:
when starting up status is being changed back to READY?

Reply:
yes in the [NotifyReady](https://github.com/weaviate/weaviate/pull/7850/files#diff-8c18e1c19c882f2c7823cb9c5eb7f0a6fe7d5cc42701a0b1bbda064013ae6232R184) which is called inside `NewShard()`

Reply:
let me run the chaos pipeline to make sure nothing breaks :D

Reply:
https://github.com/weaviate/weaviate-chaos-engineering/actions/runs/14511671089